pkg/model/manifest.go

```diff
@@ -948,13 +948,6 @@ func (manifest *Manifest) ExpandEnvVars() error {
 				return err
 			}
 		}
-		for idx, cmd := range manifest.Destroy.Commands {
```

Comment:
I think it's a good idea, maybe it might sound being extra careful but I agree that's what we need to make the product super stable and avoid regressions. This is a test that should cover the removal:

```go
{
    name: "does not expand vars in destroy command",
    envs: map[string]string{
        "TEST_VAR": "test",
    },
    manifest: &Manifest{
        Destroy: &DestroyInfo{
            Image: "",
            Commands: []DeployCommand{
                {
                    Name: "test",
                    Command: `TEST_VAR="do-not-expand-me"
echo $TEST_VAR`,
                },
            },
        },
    },
    expectedManifest: &Manifest{
        Destroy: &DestroyInfo{
            Image: "",
            Commands: []DeployCommand{
                {
                    Name: "test",
                    Command: `TEST_VAR="do-not-expand-me"
echo $TEST_VAR`,
                },
            },
        },
    },
```